### PR TITLE
add some small optimize

### PR DESCRIPTION
**Pull request by lixiaojiee**
## What is the purpose of the change

1、modify some log describe
2、use java8 lambda expression 

## Brief changelog

/dubbo-config-api/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java
/dubbo-config-api/src/main/java/org/apache/dubbo/config/ReferenceConfig.java
/dubbo-config-api/src/main/java/org/apache/dubbo/config/ServiceConfig.java
/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java

## Verifying this change

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-dubbo/tree/master/dubbo-test).
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by carryxyh** [comment]
lgtm

File: dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java [hunk 1]
```diff
@@ -186,7 +186,7 @@ public void doSaveProperties(long version) {
             } else {
                 registryCacheExecutor.execute(new SaveProperties(lastCacheChanged.incrementAndGet()));
             }
-            logger.warn("Failed to save registry store file, cause: " + e.getMessage(), e);
+            logger.warn("Failed to save registry cache file, cause: " + e.getMessage(), e);
```
Comment (khanimteyaz) [inline dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java]: I think store was more appropriate as, it is saving in file and it is persistance. What do you say?

Comment (lixiaojiee) [reply dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java]: The file contains a copy of the Dubbo data in zk. Generally speaking, this copy is also a kind of cache,so I change the "store" to "cache"

Comment (carryxyh) [reply dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java]: Hi, @lixiaojiee @khanimteyaz 
Seems like both of u are good.
The difference is not very large. I will deal with it when we reach an agreement.

Comment (khanimteyaz) [reply dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java]: yes..I am fine with this.

Comment (lixiaojiee) [reply dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java]: thanks

**Pull request merged**